olp-cpp-sdk-core/src/cache/DiskCache.cpp

```diff
@@ -43,6 +43,7 @@ namespace cache {
 namespace {
 constexpr auto kLogTag = "DiskCache";
 constexpr auto kLevelDbLostFolder = "/lost";
+constexpr auto kMinL0Files = 4;
```

Comment:
```suggestion
constexpr auto kMaxL0Files = 4;
```
Depends on how you look at it, but for our use-case the glass it is max L0 files.